assets/ui.css

```diff
@@ -427,6 +427,27 @@
   background: url("https://respec.org/xref/loader.gif") no-repeat center;
 }
 
+.respec-dfn-list li span {
+  margin-left: 0.5em;
+  padding: 0.1em;
+  text-align: center;
+  white-space: nowrap;
+  font-size: 90%;
+  border-radius: 0.2em;
+}
+
+.respec-dfn-list li span.exported {
+  background: #fcfbee;
```

Comment:
![image](https://user-images.githubusercontent.com/3396686/72333621-6eb2e680-36ff-11ea-9edd-3157fa9ab3f4.png)

Hard to tell the difference between unused in no-green/no-red accessibility emulation mode, we might want to adjust colors for better accessibility.

Reply:
I found `#d1edfd` to be a good fit for background for `span.exported`. WDYT?

Reply:
Wow. GitHub shows color preview in comments!

Reply:
Yes, seems the following works:

```css
.respec-dfn-list li span.unused {
	background: #d1edfd;
	color: #040b1c;
	box-shadow: 0 0 0 0.125em #1ca5f940;
}
```

Reply:
Can the selector be simplified? I think `.unused` is enough, or maybe `.unused-dfn` for less ambiguity.

Reply:
I think `.respec-dfn-list` based scoping is good idea (makes it easy to keep related code together). We can remove the `li span` part.